src/mixins/clipboard.js

```diff
@@ -6,8 +6,16 @@ export default {
   mixins: [alertMixin],
   methods: {
     copyToClipboard(text, { alertMessage = this.$t('clipboard_copied') } = {}) {
-      copyToClipboard(text);
-      this.alertPromptSuccess(alertMessage);
+      if (navigator.share) {
+        navigator.share({
+          title: 'Just title',
+          text: 'Just text',
```

Comment:
-`copyToClipboard` is a generic method, I think we should check the text is a URL first, it might be plain text
- Should pass `title`/`text` from function params, not hardcoded `Just title` or `Just text`

Reply:
but i check `copyToClipboard` only use here
`  copyURLPath(path, options) {
      const host = `${window.location.protocol}//${window.location.host}`;
      const url = `${host}${path}`;
      this.copyToClipboard(url, options);
    },`
so input text is a URL... Or worry about using it elsewhere in the future？

Reply:
Yes, this function might use elsewhere in the future